pegasus/sites.v3/code.org/views/csf_congrats_certificate.haml

```diff
@@ -0,0 +1,86 @@
+%script{:src=>"/js/jquery.placeholder.js"}
+%script{:src=>"/js/base64.min.js"}
+
+:javascript
+
+  function showEmailError(error)
+  {
+    $("input#email").toggleClass("input_error", error);
+    $("#email_error").toggle(error);
+    $("#email_okay").toggle(!error);
+  }
```

Comment:
We decided a while back that new javascript code should not be added to haml files (and javascript should be extracted from existing haml when possible). Can you please extract this into a separate `.js` file?